README.md

```diff
@@ -1,133 +1,150 @@
-
 <a name="readme-top"></a>
+
+
+
+<!-- TABLE OF CONTENTS -->
+
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Install](#install)
   - [Usage](#usage)
+  - [Run tests](#run-tests)
   - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
-- [⭐ Show your support](#support)
+- [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
-# 📖 HELLO MICROVERSE <a name="about-project"></a>
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
-Portfolio is my second microverse project. I deigned a mobile first portfolio site using Html and CSS flex styling
+A code setup designed to spot html and css errors.
```

Comment:
- [ ] Please ensure that your README contains information pertaining to this Project so that when users visit your repository, they aren't misled by the information in your README :wink: